content/en/docs/reference/_index.md

```diff
@@ -20,6 +20,7 @@ This section of the Kubernetes documentation contains references.
 
 * [Kubernetes API Overview](/docs/reference/using-api/api-overview/) - Overview of the API for Kubernetes.
 * Kubernetes API Versions
+  * [1.13](/docs/reference/generated/kubernetes-api/v1.13/)
   * [1.12](/docs/reference/generated/kubernetes-api/v1.12/)
```

Comment:
@tengqm, will this page eventually have the recent updates to all pages, such as the feedback section?
Perhaps this happens once this branch is merged to dev-1.13 and then master?

Reply:
yes, @kbhawkey, when new changes made to 1.13 api, the docs should be updated as well. The sad thing is that we don't have an auto-triggering mechanism for this. The best workaround I can think of is to have the upstream code changes mention sig-docs.